session/manager.go

```diff
@@ -58,7 +58,7 @@ func (manager *manager) Create(peerID identity.Identity) (sessionInstance Sessio
 	return sessionInstance, nil
 }
 
-// FindSession returns underlying session instance
+// FindClientSession returns underlying session instance
```

Comment:
Comment and function name does not match.

Reply:
Fixed